prototype/jurisdictions/certificate.AU.xml

```diff
@@ -127,216 +115,102 @@
+                    </option>
+                </radioset>
+            </question>
+            <if test="this.contentRights() === 'norights'">
+                <question id="explicitWaiver" display="The content has been">
+                    <label>Is the content of the data marked as public domain?</label>
+                    <yesno yes="marked as public domain"/>
+                    <help>Content can be marked as public domain using the <a href="https://creativecommons.org/about/pdm/">Creative Commons Public Domain Mark</a>. This helps people know that it can be freely reused.</help>
+                    <requirement level="standard">You should <strong>mark public domain content as public domain</strong> so that people know they can reuse it.</requirement>
+                </question>
+            </if>
+            <if test="this.contentRights() === 'samerights'">
+                <question id="contentLicence" display="The content is available under">
+                    <label>Under which licence or waiver can others reuse content?</label>
+                    <select required="required">
+                        <option value="cc-by">Creative Commons Attribution</option>
+                        <option value="cc-by-sa">Creative Commons Attribution Share-Alike</option>
+                        <option value="cc-zero">Creative Commons CCZero</option>
+                        <option value="other" display="">Other...</option>
+                    </select>
+                    <help>Remember that whoever spends intellectual effort creating content automatically gets rights over it but creative content does not include facts. So people need a waiver or a licence which proves that they can use the content and explains how they can do that legally. We list the most common open licenses and waivers here.</help>
+                </question>
+                <if test="this.contentLicence() === 'other'">
+                    <question id="otherContentLicenceName" display="The content is available under">
+                        <label>What's the name of the licence or waiver?</label>
+                        <input required="required" placeholder="Licence or Waiver Name"/>
+                        <help>We need the name of your licence or waiver so people can see it on your Open Data Certificate.</help>
+                    </question>
+                    <question id="otherContentLicenceURL" display="The content licence is at">
+                        <label>Where is the licence or waiver?</label>
+                        <input type="url" required="required" placeholder="Licence or waiver URL"/>
+                        <help>Provide a URL to the licence or waiver, so people can see it on your Open Data Certificate and check that it's publicly available.</help>
+                    </question>
+                    <question id="otherContentLicenceOpen">
```

Comment:
@Stephen-Gates I don't think, technically, this question (whether the licence is open) should apply if it's a waiver.